src/hermes/commands/deposit/invenio.py

```diff
@@ -553,8 +553,10 @@ def _get_license_identifier(ctx: CodeMetaContext, license_api_url: str):
 
     parsed_url = urlparse(license_url)
     url_path = parsed_url.path.rstrip("/")
-    license_id = url_path.split("/")[-1]
+    license_id = url_path.split("/")[-1].lower()
 
+    print(f"DEBUG: License is said to be {license_url} -> {license_id}")
+    
```

Comment:
**todo:** When did we start to use `printf` debugging in HERMES? This is just a joke I don't get, right?